tests/performance_tests/test_memory_usage.py

```diff
@@ -39,8 +39,8 @@ def poly_template(monkeypatch):
     yield folder
 
 
-@pytest.mark.flaky(reruns=5)
-@pytest.mark.limit_memory("130 MB")
+# @pytest.mark.flaky(reruns=5)
```

Comment:
Does not need flaky marker anymore?

Reply:
Seems not AFAIK, at least it hasn't been failing with the marker commented out